extensions/podman/packages/extension/src/utils/powershell.ts

```diff
@@ -32,21 +32,28 @@ export interface PowerShellClient {
 const PowerShell5Exe = 'powershell.exe';
 class PowerShell5Client implements PowerShellClient {
   async isVirtualMachineAvailable(): Promise<boolean> {
+    const telemetry: Record<string, unknown> = {};
     try {
       // set CurrentUICulture to force output in english
-      const { stdout: res } = await extensionApi.process.exec(
+      const res = await extensionApi.process.exec(
         PowerShell5Exe,
         [
           '[System.Console]::OutputEncoding = [System.Text.Encoding]::Unicode; (Get-WmiObject -Query "Select * from Win32_OptionalFeature where InstallState = \'1\'").Name | select-string VirtualMachinePlatform',
         ],
         { encoding: 'utf16le' },
       );
-      if (res.indexOf('VirtualMachinePlatform') >= 0) {
+      telemetry['command'] = res.command;
+      telemetry['stdout'] = res.stdout;
+      telemetry['stderr'] = res.stderr;
+      if (res.stdout.indexOf('VirtualMachinePlatform') >= 0) {
         return true;
       }
     } catch (err) {
       // ignore error, this means that VirtualMachinePlatform not enabled
+      telemetry['error'] = err;
```

Comment:
When the process exit with non-zero exit code, the exception raised is `ResultError` it has some field that would be nice to extract like `exitCode`, `stderr` and `stdin`

https://github.com/podman-desktop/podman-desktop/blob/1a08b7ba22a9a29bf6963ad66c2efa9de0c8e0b9/packages/extension-api/src/extension-api.d.ts#L4407